package.json

```diff
@@ -4,11 +4,13 @@
   "private": true,
   "license": "Apache-2.0",
   "workspaces": [
+    "packages/@guardian/*",
     "src/core/*",
     "src/core/components/*",
-    "packages/@guardian/*"
+    "scripts/analysis"
```

Comment:
What is the advantage of this script being in a workspace? Is it just to isolate the dependencies?

Reply:
Yeah, that's why I did it. I guess the alternative is to install them as dev dependencies via the top-level `package.json`? Happy to do it that way if you'd prefer.

Reply:
Another alternative would be to make the whole `scripts` directory the workspace, rather than the nested `analysis` directory?

Reply:
No, I like this approach, it makes it clearer where things gets used. I say let's go with it 😄